game.js

```diff
@@ -1,3 +1,39 @@
 // Write all your code here
 // Replace the contents of this file with your own code
-console.log('Hello, world!');
+const choices = ["ROCK","SCISSOR","PAPER"];
```

Comment:
In the original game, choices are rock, paper, and scissors, but you used scissor it can be confusing for players.